src/auto-evo/simulation/food_source/MarineSnowFoodSource.cs

```diff
@@ -36,7 +39,9 @@ public override float FitnessScore(Species species, SimulationCache simulationCa
             score *= Constants.AUTO_EVO_CHUNK_LEAK_MULTIPLIER;
         }
 
-        score /= energyBalance.FinalBalanceStationary;
+        // I don't think it makes sense here to penalize species with positive ATP balance (with higher balance
+        // reducing the available energy), so that divide by final balance stationary here is gone - hhyyrylainen
+        score *= Constants.AUTO_EVO_MARINE_SNOW_BASE_MULTIPLIER;
```

Comment:
two things here:

- food source fitness scores should only be used to compare to other scores from the same source, so I hope that a constant multiplier is useless (or I designed this wrong)
- This was intended to be a penalty for higher osmoregulation costs, so scavengers don't get bigger than they need to be. Combined with the bonus for energy balance on line 30, I think you will start to see a bias towards progressively larger species with smaller and smaller populations. Ironically this might lead to patches periodically dieing off, but I wouldn't know without a lot of testing.